services/src/main/java/org/broadleafcommerce/payment/service/gateway/DefaultPayPalCheckoutTransactionService.java

```diff
@@ -114,20 +118,27 @@ public PaymentResponse authorize(PaymentRequest paymentRequest) {
                 Amount amount = transaction.getAmount();
                 Authorization authorization = getAuthorization(transaction);
 
+                String updateTime = authorization.getUpdateTime();
+
                 paymentResponse
```

Comment:
Use the `DefaultPaymentResponseUtil` to build & populate the PaymentResponse based on the PaymentRequest. From there, we should populate additional properties.